CHANGELOG.md

```diff
@@ -5,13 +5,22 @@
 **Breaking Changes**:
 
 - The `sentry_options_set_logger` function now accepts a `userdata` parameter.
-- The `name` parameter of `sentry_options_add_attachment(w)` was removed, it will
-  now be inferred from the filename of `path`.
+- The `name` parameter of `sentry_options_add_attachment(w)` was removed, it will now be inferred from the filename of `path`.
```

Comment:
For the breaking changes, what if we add little code examples for the migration or new usage? I think this could help in upgrading, or do you feel that this would be too much?